nautobot/virtualization/models.py

```diff
@@ -290,6 +291,12 @@ class VirtualMachine(PrimaryModel, ConfigContextModel, StatusModel):
         "memory",
         "disk",
     ]
+    # 2.0 TODO: Make this go away when we assert filterset/filterform parity.
+    # FilterSet fields that are custom on FilterForm that we don't want in
+    # DynamicGroup UI edit form.
```

Comment:
Do not want, because they are already there. I clarified the comment. This is a one-off for now, since this wasn't necessary for `Device`. If we end up expanding Dynamic Groups to other models before we refactor everything, this will no longer be a one-off.